p2p/memp2p/memp2p_test.go

```diff
@@ -141,80 +147,74 @@ func TestBroadcastingMessages(t *testing.T) {
 }
 
 func TestConnectivityAndTopics(t *testing.T) {
-	network, err := memp2p.NewNetwork()
-	assert.Nil(t, err)
+	network, _ := memp2p.NewNetwork()
 	network.LogMessages = true
 
-	peer1, err := memp2p.NewMessenger(network)
-	assert.Nil(t, err)
-	peer2, err := memp2p.NewMessenger(network)
-	assert.Nil(t, err)
-	peer3, err := memp2p.NewMessenger(network)
-	assert.Nil(t, err)
-	peer4, err := memp2p.NewMessenger(network)
-	assert.Nil(t, err)
-
-	// All peers listen to the topic "rocket"
-	err = peer1.CreateTopic("rocket", false)
-	assert.Nil(t, err)
-	err = peer1.RegisterMessageProcessor("rocket", mock.NewMockMessageProcessor(peer1.ID()))
-	assert.Nil(t, err)
-	err = peer2.CreateTopic("rocket", false)
-	assert.Nil(t, err)
-	err = peer2.RegisterMessageProcessor("rocket", mock.NewMockMessageProcessor(peer2.ID()))
-	assert.Nil(t, err)
-	err = peer3.CreateTopic("rocket", false)
-	assert.Nil(t, err)
-	err = peer3.RegisterMessageProcessor("rocket", mock.NewMockMessageProcessor(peer3.ID()))
-	assert.Nil(t, err)
-	err = peer4.CreateTopic("rocket", false)
-	assert.Nil(t, err)
-	err = peer4.RegisterMessageProcessor("rocket", mock.NewMockMessageProcessor(peer4.ID()))
-	assert.Nil(t, err)
+	// Create 4 peers on the network, all listening to the topic "rocket".
+	for i := 1; i <= 4; i++ {
+		peer, _ := memp2p.NewMessenger(network)
+		peer.CreateTopic("rocket", false)
+		processor := mock.NewMockMessageProcessor(peer.ID())
+		peer.RegisterMessageProcessor("rocket", processor)
```

Comment:
if return code is not needed from RegisterMessageProcessor, explicitly show this with
 _ = peer.RegisterMessageProcessor...

Reply:
Changed.